assets/js/app.js

```diff
@@ -0,0 +1,165 @@
+const displayController = (() => {
+  const domBoard = document.getElementById('board');
+  const renderBoard = (board) => {
+    let renderedBoard = '';
+
+    for (let i = 0; i < board.length; i += 1) {
+      for (let j = 0; j < board[i].length; j += 1) {
+        const cellValue = board[i][j];
+        renderedBoard += ` <div class="cell" onclick='game.applyMove(${i},${j})'>${cellValue}</div>`;
+      }
+    }
+    domBoard.innerHTML = renderedBoard;
+  };
+
+  return {
+    renderBoard
+  };
+})();
```

Comment:
Let's separate the DOM from application logic, you can create another file for DOM.

![image](https://user-images.githubusercontent.com/31897434/73420341-70e18b80-4322-11ea-840a-fa14e04b592b.png)

https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59623451/